samples/browser/src/geckoNightly/java/org/mozilla/samples/browser/Components.kt

```diff
@@ -19,7 +19,7 @@ import org.mozilla.geckoview.GeckoRuntimeSettings
 class Components(private val applicationContext: Context) : DefaultComponents(applicationContext) {
     private val runtime by lazy {
         // Allow for exfiltrating Gecko metrics through the Glean SDK.
-        val builder = GeckoRuntimeSettings.Builder()
+        val builder = GeckoRuntimeSettings.Builder().aboutConfigEnabled(true)
```

Comment:
Maybe a stupid question - why didn't we want `about:config` allowed before?

Reply:
I'm not sure 😄,  maybe no body need it before.